app/src/main/java/net/skyscanner/backpack/demo/compose/DialogStory.kt

```diff
@@ -187,6 +189,7 @@ fun FlareDialogExample(onDismiss: () -> Unit = {}) {
       painter = painterResource(R.drawable.canadian_rockies_canada),
       contentDescription = stringResource(R.string.image_rockies_content_description),
       contentScale = ContentScale.Crop,
+      modifier = Modifier.height(264.dp),
```

Comment:
Set a size modifier here. This matches figma better and also the dynamic size caused some issues when taking screenshots directly with compose